packages/trpc/server/routers/viewer.tsx

```diff
@@ -1167,6 +1172,33 @@ const loggedInViewerRouter = router({
         });
       }
     }),
+  getDownloadLinkOfCalVideoRecordings: authedProcedure
+    .input(
+      z.object({
+        recordingId: z.string(),
+      })
+    )
+    .query(async ({ input, ctx }) => {
+      const { recordingId } = input;
+      const { session } = ctx;
+
+      const isDownloadAllowed = IS_SELF_HOSTED || session.user.belongsToActiveTeam;
+
+      if (!isDownloadAllowed) {
+        throw new TRPCError({
+          code: "FORBIDDEN",
+        });
+      }
+
+      try {
+        const res = await getDownloadLinkOfCalVideoByRecordingId(recordingId);
+        return res;
+      } catch (err) {
+        throw new TRPCError({
+          code: "BAD_REQUEST",
+        });
```

Comment:
Should we provide more detail if we ever hit this? err.message maybe? (Not sure of the error thrown here could be differnt)